src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -1,21 +1,26 @@
 <template>
   <div
     class="ec-checkbox"
+    :class="$attrs.class"
+    :style="$attrs.style"
     :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-checkbox` : 'ec-checkbox'"
   >
     <input
-      v-bind="$attrs"
-      :id="id"
+      v-bind="{
+        ...$attrs,
+        style: null,
+        class: 'ec-checkbox__input',
+        id: id,
+        'aria-describedby': errorId,
+        disabled: disabled,
+        'data-test': 'ec-checkbox__input',
+        type: 'checkbox'
+      }"
       ref="checkboxInput"
       v-model="inputModel"
-      :aria-describedby="errorId"
-      type="checkbox"
-      class="ec-checkbox__input"
-      :disabled="disabled"
-      data-test="ec-checkbox__input"
       @focus="inputIsFocused = true"
       @blur="inputIsFocused = false"
-      v-on="{ ...$listeners, 'update:modelValue': null }"
+      v-on="{ 'update:modelValue': null }"
```

Comment:
we shouldn't need this one after enabling MODE: 3. The problem with Vue 2 was that `<ec-checkbox v-model="blah">` made the `update:modelValue` event appearing in `$listeners` so I had to remove it from there so it didn't get attached to the child input. After removing $listeners, it's safe to remove this line, because we're now binding `$attrs` only and they do not contain the `update:modelValue` event because is declared in `defineEmits` array.